src/imitation/scripts/config/train_adversarial.py

```diff
@@ -242,7 +239,6 @@ def seals_swimmer():
 
 @train_adversarial_ex.named_config
 def seals_walker():
-    locals().update(**MUJOCO_SHARED_LOCALS)
```

Comment:
Careful! While the others were commented out, this one was not. Are you sure you want to remove this line?

Reply:
Yes, this line was vestigial as the `MUJOCO_SHARED_LOCALS` are anyways getting updated with the config below it.